.kitchen.ec2.yml

```diff
@@ -1,69 +1,49 @@
-<%  # This file contains ERB-interpreted YAML directives for testkitchen.
-    # Secrets are read from the local execution ENV into this file, and
-    # passed to the Chef tests via node attributes applied to the test suites.
-
-# To keep the YAML as simple as possible, some values are computed here with defaults set so kitchen will run without
-# values being set in ENV - at least as far as listing or running on the vagrant driver
-AWS_ACCESS_KEY_ID = ENV['AWS_BLUE_ACCESS_KEY_ID'] ||'(ENTER_AWS_ACCESS_KEY_ID)'
-AWS_ACCESS_KEY_SECRET = ENV['AWS_BLUE_SECRET_ACCESS_KEY_ID'] || '(ENTER_AWS_SECRET_ACCESS_KEY)'
+<%
+# To keep the YAML as simple as possible, some values are computed here
+AWS_ACCESS_KEY_ID = ENV['AWS_ACCESS_KEY_ID'] ||'(ENTER_ACCESS_KEY_ID)'
+AWS_ACCESS_KEY_SECRET = ENV['AWS_SECRET_KEY'] || '(ENTER_AWS_SECRET_ID)'
 AWS_SSH_KEY_ID = ENV['AWS_SSH_KEY_ID'] || 'default'
 AWS_SSH_KEYPATH = File.expand_path(ENV['AWS_SSH_KEY_PATH'] || "#{ENV['HOME']}/.ssh/id_rsa")
-
-# support function, needed for inserting raw RSA private key data into YAML
-def yaml_section(hash, indentation)
-  YAML.dump(hash).sub(/\A---\s/, '').split("\n").map do |l|
-    (" " * indentation) + l
-  end.join("\n")
-end
 %>
 ---
 driver:
   name: ec2
-  aws_access_key_id: <%= ENV['AWS_BLUE_ACCESS_KEY_ID'] ||
-                        '(ENTER_AWS_ACCESS_KEY_ID)' %>
-  aws_secret_access_key: <%= ENV['AWS_BLUE_SECRET_ACCESS_KEY_ID'] ||
-                            'ENTER_AWS_SECRET_ACCESS_KEY_ID' %>
-  aws_ssh_key_id: <%= ENV['AWS_SSH_KEY_ID'] || 'default' %>
-  ssh_key: <%= File.expand_path(ENV['AWS_SSH_KEY_PATH'] ||
-                             "#{ENV['HOME']}/.ssh/id_rsa") %>
+  aws_access_key_id: <%= AWS_ACCESS_KEY_ID %>
+  aws_secret_access_key: <%= AWS_ACCESS_KEY_SECRET %>
+  aws_ssh_key_id: <%= AWS_SSH_KEY_ID %>
+  ssh_key: <%= AWS_SSH_KEYPATH %>
   region: us-east-1
   availability_zone: us-east-1c
   flavor_id: <%= ENV['AWS_FLAVOR_ID'] || '(ENTER_AWS_FLAVOR_ID)' %>
```

Comment:
I'm saying all key values are read from environment variables into constants in block at top of file, but not flavor_id/user. Unless those are prompted for when you run test-kitchen? Just looking for consistency if they are ultimately treated the same way.